### ISSUE TITLE
Droid4: Include patch to work around thermal problems

### ISSUE THREAD

**Issue by MerlijnWajer** [maemo-leste/bugtracker#490]
https://lore.kernel.org/linux-omap/20201029100335.27665-1-peter.ujfalusi@ti.com/

**Comment by parazyd** [comment]
What's the status of this?

**Comment by parazyd** [comment]
Closing per tmlind's reply

```
>10:03 <tmlind> parazyd: no need to, that's already in v5.10. i just posted some thermal patches to the lists, and merged them for testing into droid4-pending-v5.10
```
